packages/synapse-react-client/src/components/download_list/DownloadConfirmationUI.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useMemo } from 'react'
+import { Alert, AlertProps, Box, Button, Typography } from '@mui/material'
+import { useSynapseContext } from '../../utils'
+import SignInButton from '../SignInButton'
+import DownloadDetails from './DownloadDetails'
+import { useGetDownloadListStatistics } from '../../synapse-queries'
+
+enum StatusEnum {
+  LOADING_INFO,
+  PROCESSING,
+  INFO,
+  INFO_ITEMS_IN_LIST,
+  SIGNED_OUT,
+}
+
+export type DownloadConfirmationUIProps = {
+  onAddToDownloadCart: () => void
+  fileCount: number
+  fileSize?: number
+  onCancel: () => void
+  isLoadingStats?: boolean
+  isAddingToDownloadCart?: boolean
+}
+
+type UiStateDictionary = {
```

Comment:
No changes to rendered UI states

Reply:
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/0bcab9bf-5c8d-4ea5-87cf-3700b2ded1e1)